components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -22,33 +22,200 @@
 
 #import "TabBarIconExampleSupplemental.h"
 
+#import "MaterialAppBar.h"
 #import "MaterialButtons.h"
 #import "MaterialTabs.h"
+@import MaterialComponents.MaterialPalettes;
 
 @implementation TabBarIconExample (Supplemental)
 
 - (void)setupExampleViews {
```

Comment:
This method is kinda massive now. I might break it down into methods or delimit it with some comments.